tests/sensors/omron_d6t/test_omron_d6t.py

```diff
@@ -0,0 +1,32 @@
+from unittest import TestCase
+from mock import patch, Mock
+from sensors.sensor import Sensor
+from sensors.omron_d6t.omron_d6t import OmronD6T
+
+class TestOmronD6T(TestCase):
+    def setUp(self):
+        with patch.object(Sensor, "__init__", return_value = Mock()):
```

Comment:
We need to mock the `__init__` for `Sensor` since it does some I2C shenanigans that are not available in other environments than the Pi.

Reply:
I think it would be good if we can rely completely on pigpio for i2c and gpio operations.

pigpio provides remote connections to pi's (good for remote debugging), but also, relying solely on pigpio would make it much easier for us, as we'd only have to mock the methods provided by pigpio.

Thoughts?